packages/icons/src/index.js

```diff
@@ -118,6 +121,7 @@ export { default as moreHorizontalMobile } from './library/more-horizontal-mobil
 export { default as moreVertical } from './library/more-vertical';
 export { default as moveTo } from './library/move-to';
 export { default as navigation } from './library/navigation';
+export { default as noBug } from './library/no-bug';
```

Comment:
"bug" is the correct term. I'm wondering if there's a better name than "no-bug", though. I can't think of one — we have "link" and "unlink" as the closest equivalents. "unbug" won't work 🙈 

"remove-bug"?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32371/commits/7c8fa25e3393437ddfa10d2fbf2a10550403d02d